src/UNIV2LPOracle.t.sol

```diff
@@ -363,6 +363,68 @@ contract UNIV2LPOracleTest is DSTest {
         assertEq(rootVal, rootAltVal);
     }
 
+
+
+    uint112 max112 = type(uint112).max;
+
+    function test_seek_equivalence(
```

Comment:
Tbh I'm not sure what value this test really adds. We don't care about the relative gas usage, since avoiding silent overflows is the higher priority, and math operations are cheap enough that it won't be a big difference anyway. The calculations are also manifestly equivalent, and should be covered by other existing tests besides. Further, this test will be flaky due to overflow regimes. So, I'd advocate dropping it.